projects/1.20/assets/tetracelium/tetracelium/lang/zh_cn.json

```
@@ -61,6 +59,140 @@
     "tetra.material.mana_string.prefix": "蕴魔丝线",
     "tetra.improvement.mana_repair.name": "魔力修复",
     "tetra.improvement.mana_repair.description": "物品会随时间使用魔力修复自己",
+    "tetra.stats.manaRepair": "魔力修复",
+    "tetra.stats.manaRepair.tooltip": "物品会随时间使用魔力修复自己，修复1点耐久度消耗§e%s§r点魔力",
+    "tetra.material.ironwood": "铁木",
+    "tetra.material.ironwood.prefix": "铁木",
+    "tetra.material.feature.rootboon": "根系恩惠",
+    "tetra.material.feature.rootboon.tooltip": "用铁木制作的不同类型主要组件会获得不同魔咒\n \n§7获得材料魔咒需要物品已有魔咒与其兼容",
+    "tetra.material.feature.rootboon.tooltip_extended": "对应的魔咒的种类见下：\n §7- 同时为钝器/刃器与方块破坏工具：§e时运 I\n §7- 方块破坏工具：§e效率 I\n §7- 可损坏：§e耐久 I",
+    "tetra.material.steeleaf": "钢叶",
+    "tetra.material.steeleaf.prefix": "钢叶",
+    "tetra.material.feature.leafboon": "绿叶恩惠",
+    "tetra.material.feature.leafboon.tooltip": "用钢叶制作的不同类型主要组件会获得不同魔咒\n \n§7获得材料魔咒需要物品已有魔咒与其兼容",
+    "tetra.material.feature.leafboon.tooltip_extended": "对应的魔咒的种类见下：\n §7- 同时为钝器/刃器与方块破坏工具：§e效率 II\n §7- 方块破坏工具：§e时运 II\n §7- 钝器/刃器：§e抢夺 II",
+    "tetra.material.fiery_ingot": "炽铁",
+    "tetra.material.fiery_ingot.prefix": "炽铁",
+    "tetra.material.feature.fiery_igniting": "灼燃",
+    "tetra.material.feature.fiery_igniting.tooltip": "用炽铁锭制作的不同槽位主要组件会获得§e灼燃§r或§e烈焰§r强化",
+    "tetra.material.feature.fiery_igniting.tooltip_extended": "手柄、剑柄，弓材等用于持握的组件会获得§e烈焰§7强化。\n剑刃刀刃，工具头部等通常用于击打目标的组件会获得§e灼燃§7强化。",
+    "tetra.material.feature.fiery_auto_smelt": "自动烧炼",
+    "tetra.material.feature.fiery_auto_smelt.tooltip": "带有§l方块破坏工具§r特征，且用炽铁锭制作的主要组件会获得§e自动烧炼§r强化",
+    "tetra.improvement.fiery_igniting.name": "灼燃",
+    "tetra.improvement.fiery_igniting.description": "击中时点燃实体",
+    "tetra.stats.igniting": "灼燃",
+    "tetra.stats.igniting.tooltip": "击中时使实体着火§e%s§r秒",
+    "tetra.improvement.fiery_blazing.name": "烈焰",
+    "tetra.improvement.fiery_blazing.description": "使用此物品时有概率点燃持有者",
+    "tetra.improvement.auto_smelt.name": "自动烧炼",
+    "tetra.improvement.auto_smelt.description": "破坏方块时自动烧炼掉落物",
+    "tetra.stats.autoSmelt": "自动烧炼",
+    "tetra.stats.autoSmelt.tooltip": "破坏方块时自动烧炼掉落物",
+    "tetra.material.fiery_blood": "炽热的血液",

```

同步一下格式？

```suggestion
    "tetra.material.fiery_blood": "炽热之血",
```
具体看强调的是血液还是炽热了

---

这个是暮色里的物品名称